recipes/jupyterlab-lsp/meta.yaml

```diff
@@ -0,0 +1,115 @@
+{% set name = "jupyter-lsp" %}
+{% set ext_name = "@krassowski/jupyterlab-lsp" %}
+{% set version = "0.8.0" %}
+{% set build_number = "0" %}
+
+{% set py_spec = ">=3.5" %}
+{% set lab_spec = ">=2,<3.0.0a0" %}
+{% set ext_spec = "@^1.0.0" %}
+
+# List of jupyter_lsp py.tests ran for each known servers
+{% set server_tests = ["serverextension", "start_known", "listeners"] %}
+
+package:
+  name: {{ name }}-suite  # To avoid issue https://github.com/conda/conda-build/issues/3933
```

Comment:
A more common suffix is `-meta`

Reply:
This name doesn't matter if you use `extra: feedstock-name: jupyterlab-lsp` and the feedstock will be named `jupyterlab-lsp-feedstock`

Reply:
Thanks for the suggestion and tip @xhochy and @isuruf